go.mod

```diff
@@ -1,31 +1,25 @@
 module github.com/apache/apisix-ingress-controller
 
-go 1.13
+go 1.16
 
 require (
-	github.com/fsnotify/fsnotify v1.5.0 // indirect
-	github.com/gin-gonic/gin v1.6.3
-	github.com/google/uuid v1.2.0 // indirect
-	github.com/hashicorp/go-memdb v1.0.4
-	github.com/hashicorp/go-multierror v1.1.0
-	github.com/hashicorp/golang-lru v0.5.3 // indirect
-	github.com/imdario/mergo v0.3.11 // indirect
-	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/onsi/ginkgo v1.16.4 // indirect
-	github.com/prometheus/client_golang v1.10.0
+	github.com/gin-gonic/gin v1.7.7
+	github.com/hashicorp/go-memdb v1.3.2
+	github.com/hashicorp/go-multierror v1.1.1
+	github.com/prometheus/client_golang v1.11.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/slok/kubewebhook/v2 v2.1.0
-	github.com/spf13/cobra v1.1.1
+	github.com/slok/kubewebhook/v2 v2.2.0
+	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
 	github.com/xeipuuv/gojsonschema v1.2.0
-	go.uber.org/multierr v1.3.0
-	go.uber.org/zap v1.13.0
-	golang.org/x/net v0.0.0-20210510120150-4163338589ed
+	go.uber.org/multierr v1.7.0
+	go.uber.org/zap v1.19.1
+	golang.org/x/net v0.0.0-20211216030914-fe4d6282115f
 	golang.org/x/sys v0.0.0-20210819072135-bce67f096156 // indirect
-	golang.org/x/tools v0.1.5 // indirect
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.1
-	k8s.io/apimachinery v0.21.1
-	k8s.io/client-go v0.21.1
-	k8s.io/code-generator v0.21.1
+	k8s.io/api v0.22.4
+	k8s.io/apimachinery v0.22.4
+	k8s.io/client-go v0.22.4
```

Comment:
Need to re-run `make codegen` to see if there are any differents.

Reply:
There is no change. `make codegen` has been executed in this PR, and it has passed the verification in the test case.